encryption/ntru/ntru.yaml

```diff
@@ -1,6 +1,6 @@
 name: NTRU
+description: Based rather closely on the original NTRU from 1998 #TODO
```

Comment:
I'm unsure here

Reply:
True, but still probabilistic PKE based KEMs. Maybe take the gist from https://eprint.iacr.org/2018/1174.pdf

Reply:
It seems to me they have switched to correct and deterministic encryption for the round 2 submission, so the stuff from that paper does not apply anymore.